app/services/parse_offenders.rb

```diff
@@ -1,10 +1,31 @@
 module ParseOffenders
+  KEYS_MAPPING = {
+    noms_number: :noms_id,
+    alias_offender_id: :nomis_offender_id,
+    alias_date_of_birth: :date_of_birth,
+    alias_given_name_1: :given_name_1,
+    alias_given_name_2: :given_name_2,
+    alias_given_name_3: :given_name_3,
+    alias_surname: :surname,
+    salutation: :title,
+    alias_gender: :gender,
+    pnc_id: :pnc_number,
+    nationality_code: :nationality_code,
```

Comment:
@cesidio serves as handy documentation without having to look at one of the sample/fixture extracts.